examples/filetable.py

```diff
@@ -15,80 +16,96 @@ def filetable_to_firefly(ffclient, topdir, pattern, recursive=True):
     ffclient: firefly_client.FireflyClient
         Instance of FireflyClient connected to a Firefly server
 
-    topdir: 'str'
+    topdir: "str"
         pathname for directory to search
 
-    pattern: 'str'
-        filename pattern to search for, e.g. '*.fits'
+    pattern: "str"
+        filename pattern to search for, e.g. "*.fits"
 
     recursive: `bool`
         Search all subdirectories recursively (default=True)
 
     Returns:
     --------
-    tbl_val: 'str'
+    tbl_val: "str"
         Descriptor of table that was uploaded to the Firefly server
 
     metadict: `dict`
         Dictionary of metadata items
 
     """
-    filelist = glob(topdir+'/**/' + pattern, recursive=recursive)
-    metadict = {'datasource': 'path'}
-    with tempfile.NamedTemporaryFile(mode='w+t',
-                                     delete=False,
-                                     suffix='.csv') as fd:
+    filelist = sorted(glob(topdir + "/**/" + pattern, recursive=recursive))
```

Comment:
What was the rationale for sorting?

Reply:
Oh! I am doing a glob on NEOS detector test files, and they came back randomly. I want them lexically sorted. I know, I can always sort by the filename in the Firefly table...but then the `count` column is all random.

Do you think it is better not to sort?

Reply:
@gpdf should I make sorting an optional argument to the script? Maybe I should also do that for the `/external` that is added to the file URL?

Reply:
I think that would be a good idea, yes.

Reply:
okay, sorting is now an option!